client/state/posts/revisions/selectors.js

```diff
@@ -1,21 +1,74 @@
 /**
```

Comment:
All new selectors should ideally be added to the `client/state/selectors` directory. `selectors.js` is a legacy pattern.

https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#selectors